packages/mobile/src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -121,7 +121,7 @@ function NameAndPicture({ navigation }: Props) {
           enablesReturnKeyAutomatically={true}
           placeholder={t('fullNamePlaceholder')}
           testID={'NameEntry'}
-          multiline={true}
+          multiline={false}
```

Comment:
oh interesting, this seems was inadvertently changed during accessibility fixes.

Reply:
oh I'm not sure if this textalign top styling does anything here, is there any edge cases that needs to be tested? Maybe when the name input is really long? 
https://github.com/valora-inc/wallet/blob/029b7f6bdda17fabb4a7aae9b349f6c1e9730e38/packages/mobile/src/components/TextInput.tsx#L79